src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -371,15 +371,15 @@ public String getNewQuestionSpecificEditFormHtml() {
         
         this.initializeRubricDescriptions();
         
-        setDescription(0,0, "Initiates discussions frequently, and engages the team.");
-        setDescription(0,1, "Takes part in discussions and sometimes initiates discussions.");
-        setDescription(0,2, "Occasionally responds, but never initiates discussions.");
-        setDescription(0,3, "Rarely or never responds.");
+        setDescription(0,0, "Rarely or never responds.");
```

Comment:
Since these lines are already modified let's make them better than we found them, the code convention for Java on TEAMMATES suggests that this should be ```setDescription(0, 0, "Rarely or never responds.");``` instead of ```setDescription(0,0, "Rarely or never responds.");```